app/assets/stylesheets/playtime/wishlist-item.scss

```diff
@@ -0,0 +1,8 @@
+// Styles for wishlist item view
+.wishlist-item {
+  .staff-message,
+  .item-details,
+  .wishlist-item-tools {
+    width: 100%;
+  }
+}
```

Comment:
Can you add a new line at the end of the file?

Reply:
Done!